src/utils.ts

```diff
@@ -1405,3 +1419,32 @@ export function parsePackageVersion({ version }: { version: string }): {
   const [major, minor, patch] = version.split('.').map((n: string) => Number.parseInt(n, 10));
   return { major, minor, patch };
 }
+
+/**
+ * Fisher–Yates Shuffle
+ *
+ * Reference: https://bost.ocks.org/mike/shuffle/
+ * @param sequence - items to be shuffled
+ * @param limit - the number of ite
```

Comment:
cut off sentence, needs a note about the special significance of `0` (i.e. that it means return all, not return none)

Reply:
done